bundles/binding/org.openhab.binding.weather/src/main/java/org/openhab/binding/weather/internal/common/WeatherConfig.java

```diff
@@ -167,8 +186,7 @@ private void parseApiKey(String key, String value) throws ConfigurationException
         } else if (StringUtils.equalsIgnoreCase(keyId, "apikey2")) {
             pConfig.setApiKey2(value);
         } else {
-            throw new ConfigurationException("weather",
-                    "Unknown configuration key '" + key + "', please check your openhab.cfg!");
+            logger.debug("Unknown configuration key '{}'. Check openhab.cfg.", key);
```

Comment:
In this case, someone's added a key that's not recognized by the system, like <code>myInvalidKey=7</code>.

This shouldn't cause any problems for the binding, however, as it only examines known keys. So it's merely a nicety for the user. "Hey, we found this setting we don't recognize. It's no problem for us since we're ignoring it completely. Why's it here again?"